docs/api/desktop-capturer.md

```diff
@@ -16,6 +16,12 @@ app.whenReady().then(() => {
   const mainWindow = new BrowserWindow()
 
   session.defaultSession.setDisplayMediaRequestHandler((request, callback) => {
+    // If we should use the system picker
+    // Note: this is currently experimental
```

Comment:
```suggestion
    // If we should use the MacOS system picker
    // Note: this is currently experimental
```

Reply:
Talked with Sam about this; this could technically also be used on Linux and Windows in the future, so we don't need to cal it out as MacOS specific 😄